app/components/Form/ProjectMilestoneReportFormSummary.tsx

```diff
@@ -177,6 +181,16 @@ const ProjectMilestoneReportFormSummary: React.FC<Props> = (props) => {
             milestoneReport.milestoneFormChange
           ) as any);
 
+      const paymentFormDiffObject = !renderDiff
```

Comment:
inverting the conditional would make the code a bit more readable here (and for the other `<...>diffObject`)